working dir/skins/Default Skin reference.css

```diff
@@ -332,13 +331,6 @@
      * the control doesn't have focus or the row of a previously selected item. */
     -fx-selection-bar-non-focused: lightgrey;
 
-    /* The color to use as -fx-text-fill when painting text on top of
-     * backgrounds filled with -fx-selection-bar.
-     *
-     * TODO: this can be removed
-     */
-    -fx-selection-bar-text: -fx-text-background-color;
```

Comment:
This is basically copied Modena.css, I wouldnt mess with the code. Though its outdated by now, so it should be updated or removed.

Reply:
I want to get rid of excessive todos, and as you said its outdated anyways so it doesnt matter. I've looked into this multiple times and the constant is used absolutely nowhere

Reply:
The problem is you are modifying a file that should be read-only. It should probably be removed so I'll look the other way.